packages/client/tests/functional/transaction-stored-query/tests.ts

```diff
@@ -0,0 +1,50 @@
+import testMatrix from './_matrix'
+// @ts-ignore
+import type { PrismaClient } from './node_modules/@prisma/client'
+
+declare let prisma: PrismaClient
+
+/**
+ * Regression test for #11740 & comment
```

Comment:
Should it be in `tests/functional/issues` together with other regression tests?

Reply:
I didn't know regressions should go in there, but I can move them there, thanks for noticing. Did all of us actually put the reproductions in there during the correctness initiative?

Reply:
That's a good question, not sure about that. I personally don't have a strong opinion about it, but I thought everyone else was putting them there 😅

I think @danstarns added this folder even before the correctness initiative, if I recall correctly.

Reply:
Moved into that folder, I think keeping the folder name is useful for browsing but I prepended the issue number.